src/components/debug/PairingDashboard.tsx

```diff
@@ -1,4 +1,4 @@
-import { useState } from 'react';
+import { useState } from "react";
```

Comment:
what formatter is this coming from?

Reply:
prettier, i change to eslint and seem fixed